src/EnergyPlus/CondenserLoopTowers.cc

```diff
@@ -3251,9 +3244,12 @@ namespace CondenserLoopTowers {
             Real64 WaterFlowRatio(0.0); // tower water flow rate ratio found during model calibration
 
             if (ModelCalibrated) {
-                auto f = std::bind(&CoolingTower::residualTa, this, std::placeholders::_1, std::placeholders::_2, std::placeholders::_3);
-                General::SolveRoot(
-                    state, Acc, MaxIte, SolFla, WaterFlowRatio, f, DataPrecisionGlobals::constant_pointfive, MaxWaterFlowRateRatio, Par);
+                auto f = [&state, this](Real64 FlowRatio) {
```

Comment:
Instead of creating a 6 element array and populating it with data then accessing the data back off of it, we just capture `this` and it has everything needed.  Also, if you look at the original residual of this, you'll see that the lambda is a lot simpler looking.  This makes the logic so much easier to understand.